deepwell/config.example.toml

```diff
@@ -197,3 +197,6 @@ max-name-changes = 3
 #
 # Set to 0 to disable.
 refill-name-change-days = 90
+
+# The minimum length for a user's name in bytes.
```

Comment:
I would suggest a bit of background for this option.

```suggestion
# The minimum length for a user's name in bytes.
# A user who attempts to register must have a slug which is at least this long.
#
#  The intent is to prevent the proliferation of very short, unidentifiable usernames
# such as single Latin letters (e.g. "N"). However, we check *bytes* specifically to
# avoid Anglo-centricism, as many full names in other languages fit within two letters
# (consider CJK for instance). Single characters outside of the main ASCII block
# usually take up two or three bytes themselves, making it a non-issue for those languages. 
#
# See WJ-1122.
```